app/controllers/request-to-go-live/agreement/post.controller.js

```diff
@@ -54,7 +54,7 @@ const createZendeskMessage = opts => ` Service name: ${opts.serviceName}
  Email address: ${opts.email}
  Time: ${opts.timestamp}
  Service created at: ${opts.serviceCreated}
-`
+` + (opts.takesPaymentsOverPhone === true ? ' Will take payments over the phone: Yes' : '')
```

Comment:
so there is no need to show if the answer is no?

Reply:
yeah, it is not required as most services will not have chosen 'Yes'. Also not applicable to Stripe services.